samples/nrf9160/lwm2m_client/src/lwm2m/lwm2m_app_device.c

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Btw, why did you add `_app_` prefix in some filenames and not in others? I'd be consistent here.

Reply:
I think this was done early on, as files such as lwm2m_location and lwm2m_device already existed but didn't have the functionality we wanted. But as you mentioned in another thread the lwm2m_client_utils files should probably be updated rather than us making a local version.

Reply:
Fixed. I renamed the app_gps_event to gps_pvt_event so that it doesn't conflict with the gps_event in the gps driver, and I kept lwm2m_app_utils as is because I think it makes more sense that way. And there is also already a lwm2m_utils.h in the client utils library that could potentially cause issues.